telematic_system/telematic_apps/web_app/client/src/components/ros2_rosbag/ROS2RosbagUploadDialog.js

```diff
@@ -47,7 +47,7 @@ const ROS2RosbagUploadDialog = (props) => {
       filesInfo.push({
         filename: file.name,
         filetype: file.type,
-        datetime: file.lastModifiedDate.toLocaleString(),
+        datetime: file.lastModifiedDate?.toLocaleString(),
```

Comment:
Will this just set the datetime to undefined? What will the "created at" field in mysql be set to?

Reply:
I think if this is undefined, then the created_at will use default time now()